### PR TITLE
Fixed reasons of some events.

### PR DESCRIPTION
**Pull request by jszczepkowski**
Fixed reasons of some events: made consistent with API conventions (CamelCase).

### CODE REVIEW

**Comment by jszczepkowski** [comment]
CC @bgrant0607  @justinsb

**Comment by jszczepkowski** [comment]
Some reasons of events were not consistent with the convention: https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#events

**Comment by mikedanese** [comment]
LGTM but needs rebase.

**Comment by justinsb** [comment]
Thanks for fixing - I didn't know the conventions.  LGTM.

**Comment by mikedanese** [comment]
@justinsb you can read about them here if you are interested.

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/api-conventions.md#typical-status-properties

**Comment by jszczepkowski** [comment]
@k8s-bot test this

**Comment by mikedanese** [comment]
@k8s-bot ok to test

File: pkg/controller/service/servicecontroller.go [hunk 1]
```diff
@@ -670,8 +670,7 @@ func (s *ServiceController) lockedUpdateLoadBalancerHosts(service *api.Service,
 		return nil
 	}
 
-	message := "error updating loadbalancer with new hosts: " + err.Error()
-	s.eventRecorder.Event(service, "LoadBalancerUpdateFailed", message)
+	s.eventRecorder.Eventf(service, "LoadBalancerUpdateFailed", "Error updating load balancer with new hosts: %v", err)
```
Comment (erictune) [inline pkg/controller/service/servicecontroller.go]: Including actual hosts may make it less likely that identical events will be coalesced.  Also, the hosts were not passed to the function that had the error, so they can't really be part of the Reason.

Comment (erictune) [inline pkg/controller/service/servicecontroller.go]: @jszczepkowski

**Pull request merged**